framework/TSA/TimeSeriesAnalyzer.py

```diff
@@ -31,13 +31,28 @@ class TimeSeriesAnalyzer(utils.metaclass_insert(abc.ABCMeta, object)):
   ## define the clusterable features for this trainer.
   _features = []
 
+  def canGenerate(self) -> bool:
+    """
+    A predicate function to determine if object instance inherits from TimeSeriesGenerator.
+
+    @ Out, boolean, True if instance can generate
```

Comment:
technically the RAVEN standards still require a `@ In, None` if there's no input arguments. Applies to several new functions.